test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -58,22 +58,45 @@
 PT_EC2_NEURON_TRN1_INSTANCE_TYPE = get_ec2_instance_type(
     default="trn1.2xlarge", processor="neuron", job_type="training"
 )
+PT_EC2_NEURON_INF2_INSTANCE_TYPE = get_ec2_instance_type(
+    default="inf2.xlarge", processor="neuron", job_type="training"
```

Comment:
Why do we call this process as `Neuron` and not `Neuronx`?

Reply:
Done, though i don't think it makes any functional difference